### PR TITLE
Metadata queries and full-text search

### PR DESCRIPTION
**Pull request by ashwinkumar12345**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/sql/sql-full-text.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+Pass in your search query and the field name that you want to search against.
+
+
+```sql
+SELECT account_number, address
+FROM accounts
+WHERE MATCH_QUERY(address, 'Holmes')
+```
+
+Or,
```
Comment (aetter) [inline docs/sql/sql-full-text.md:36]: Let's be a little more formal here. "Alternate syntax:"

File: docs/sql/sql-full-text.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
+---
+layout: default
+title: Full-Text Search
+parent: SQL
+nav_order: 2
+---
+
+# Full-text search
+
+Use SQL commands for full-text search. For more information about full-text queries in Elasticsearch, see [Full-text queries](../../elasticsearch/full-text/).
```
Comment (aetter) [inline docs/sql/sql-full-text.md:10]: Probably worth mentioning here that SQL has a subset of the full-text queries available in Elasticsearch.

File: docs/sql/sql-full-text.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+| account_number | address
+:--- | :---
+1 | 880 Holmes Lane
+
+
+## Score query
+
+To return a relevance score along with every matching document, use `SCORE`, `SCOREQUERY`, or `SCORE_QUERY` functions.
+
+You need to pass in two arguments. The first is the `MATCH_QUERY` expression and the second is an optional floating point number to boost the score (default value is 1.0).
```
Comment (aetter) [inline docs/sql/sql-full-text.md:112]: . The second is an optional

File: docs/sql/sql-full-text.md [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+| account_number | address
+:--- | :---
+1 | 880 Holmes Lane
+
+
+## Multi match
+
+To search for text in multiple fields, use `MULTI_MATCH`, `MULTIMATCH`, or `MULTIMATCHQUERY` functions.
+
+For example, to search for `Dale` in either the `firstname` or `lastname` fields:
```
Comment (aetter) [inline docs/sql/sql-full-text.md:54]: For example, search for `Dale` in either

File: docs/sql/sql-full-text.md (hunk 1 above)
Comment (ashwinkumar12345) [reply docs/sql/sql-full-text.md:36]: Done.

File: docs/sql/sql-full-text.md (hunk 2 above)
Comment (ashwinkumar12345) [reply docs/sql/sql-full-text.md:10]: Done.

File: docs/sql/sql-full-text.md (hunk 3 above)
Comment (ashwinkumar12345) [reply docs/sql/sql-full-text.md:112]: Done.

File: docs/sql/sql-full-text.md (hunk 4 above)
Comment (ashwinkumar12345) [reply docs/sql/sql-full-text.md:54]: Done.

**Pull request closed without merging**